xAPI.md

```diff
@@ -3352,9 +3356,12 @@ Returns: ```200 OK```, Person Object
 
 ###### Requirements
 
-All array properties must be populated with members with the 
+* All array properties must be populated with members with the 
 same definition as the similarly named property from Agent Objects.  
 
+* Additional properties not listed here SHOULD* NOT be added to this object and each 
+property MUST occur only once.  
```

Comment:
MUST for multiplicity because this could cause chaos, nobody is doing it, most JSON implementations don't support it and whilst the JSON spec doesn't specifically forbid it, it doesn't explicitly allow it either. 

SHOULD* for additional properties for the same reason as statements above. I'd be ok with a MUST too though.